content/02.main-text.md

```diff
@@ -193,6 +195,21 @@ Additional research studies in progress are being authored using Manubot, spanni
 Manubot is also being used for documents beyond traditional journal publications, such as [grant proposals](https://greenelab.github.io/manufund-2018/), [progress reports](https://greenelab.github.io/czi-hca-report/), [undergraduate research reports](https://zietzm.github.io/Vagelos2017/) [@doi:10.6084/m9.figshare.5346577], [literature reviews](https://slochower.github.io/synthetic-motor-literature/), and lab notebooks.
 Finally, manuscripts written with other authoring systems have been successfully ported to Manubot, including the [Bitcoin Whitepaper](https://git.dhimmel.com/bitcoin-whitepaper/) [@tag:steemit-post] and [Project Rephetio manuscript](https://git.dhimmel.com/rephetio-manuscript/) [@doi:10.7554/eLife.26726].
 
+### Citation utility
+
+To make citation-by-identifier easily usable outside of Manubot manuscripts, we created the `manubot cite` command line utility, available as a [Python package](https://pypi.org/project/manubot/).
```

Comment:
I like the PyPI link here, because if someone wants to install the utility, that is the most useful link.

I am now not content with the previous statement:

> Manubot is a system for writing scholarly manuscripts via GitHub that is built upon our Python package of the same name.

Alone this sentences is fine, but we don't mention the Rootstock template anywhere. I am thinking we ought to mention somewhere that users currently setup a manuscript based on our template repository.

Reply:
Do you think we should add a paragraph introducing Rootstock?

Reply:
Currently we have
> An [example repository](https://github.com/greenelab/manubot-rootstock) demonstrates Manubot’s features and serves as a template for users to write their own manuscript with Manubot.

We could say more, but would it end up being the technical steps to using the rootstock?

Reply:
If we wanted to add a bit more about Rootstock, I would prefer it be non-technical, e.g., Rootstock is a way for non-technical folks to get started using Manubot. Clone the repository, follow the instructions, edit some Markdown, and CI will automatically render the manuscript for you. It's a complement and contrast to the power and utility of the command line `manubot` utilities.

On consideration, maybe I mean "less technical" not "non-technical" because, of course, you still do need to know `git` and you have to do some Travis stuff. But consider the case of collaborators. If someone has already setup a Rootstock repository, then all a new user has to do is edit some Markdown, possibly even in the GitHub web interface, and Manubot will build the manuscript automatically. So I see Rootstock as a way to make it easier for people to take advantage of using Manubot, even if they don't have the facility to work with the command line.

Reply:
Yes, I would support adding more along those lines.  I'd say that some technical expertise is required to create a new manuscript using the rootstock based on the user reports we've seen.  But the template and step-by-step instructions make it _much_ easier than trying to set up something comparable independently.

As you note, once it is set up there is much less technical expertise required for collaborators.  It does help if the manuscript maintainer understands git well enough to help resolve workflow issues and conflicts.

Should we move this discussion to a new issue?